### PR TITLE
PBM-755 Added physical backups description into docs

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/index.rst
	modified:   source/initial-setup.rst
	modified:   source/installation.rst
	modified:   source/intro.rst
	modified:   source/pbm-commands.rst
	modified:   source/point-in-time-recovery.rst
	modified:   source/running.rst
        new file: source/backup-types.rst

### CODE REVIEW

**Review by spron-in — APPROVED** [review]

File: source/running.rst [hunk 1]
```diff
@@ -285,6 +320,10 @@ Viewing backup logs
 
 As of version 1.4.0, you can see the logs from all ``pbm-agents`` in your MongoDB environment using ``pbm CLI``. This reduces time for finding required information when troubleshooting issues.
 
+.. note::
+
+   Physical backup information is not available in pbm logs.
```
Comment (sandraromanchenko) [inline source/running.rst:325]: It's partially true - it's possible to check logs about physical backup via 'pbm logs', but on the other hand info about restore from physical backup isn't accessible using the mentioned command. Should we make a note more specific?

File: source/index.rst [hunk 2]
```diff
@@ -6,12 +6,17 @@
...
+As of version 1.7.0, |pbm| supports both physical and logical backups and restores. :ref:`pitr` is currently supported only for logical backups.
 
-.. note::
+.. rubric:: Supported MongoDB versions
+
+|PBM| is compatible with the following MongoDB versions:
+
+* For *logical* backups - `Percona Server for MongoDB <https://www.percona.com/software/mongo-database/percona-server-for-mongodb>`_ and MongoDB Community v4.0 and higher with `MongoDB Replication <https://docs.mongodb.com/manual/replication/>`_ enabled.
+
+* For *physical* backups - `Percona Server for MongoDB <https://www.percona.com/software/mongo-database/percona-server-for-mongodb>`_ or higher with `MongoDB Replication <https://docs.mongodb.com/manual/replication/>`_ enabled.
```
Comment (sandraromanchenko) [inline source/index.rst:17]: There is no actual version of PSMDB defined, is it correct?

Comment (nastena1606) [reply source/index.rst:17]: Good point, thank you !

File: source/running.rst (hunk 1 above)
Comment (nastena1606) [reply source/running.rst:325]: Amended

File: source/index.rst [hunk 3]
```diff
@@ -6,12 +6,17 @@
...
+As of version 1.7.0, |pbm| supports both physical and logical backups and restores. :ref:`pitr` is currently supported only for logical backups.
 
-.. note::
+.. rubric:: Supported MongoDB versions
+
+|PBM| is compatible with the following MongoDB versions:
+
+* For *logical* backups - `Percona Server for MongoDB <https://www.percona.com/software/mongo-database/percona-server-for-mongodb>`_ and MongoDB Community v4.0 and higher with `MongoDB Replication <https://docs.mongodb.com/manual/replication/>`_ enabled.
+
+* For *physical* backups - `Percona Server for MongoDB <https://www.percona.com/software/mongo-database/percona-server-for-mongodb>`_ v4.2 or higher with `MongoDB Replication <https://docs.mongodb.com/manual/replication/>`_ enabled.
```
Comment (sandraromanchenko) [inline source/index.rst:17]: Not good enough XD 
Backup cursor was introduced in 4.2.15-16, 4.4.6-8, 5.0+ physical backups won't work with versions which are lower

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**